deploy/charts/rook-ceph-cluster/values.yaml

```diff
@@ -53,7 +53,7 @@ monitoring:
     annotations: {}
 
 # If true, create & use PSP resources. Set this to the same value as the rook-ceph chart.
-pspEnable: true
+pspEnable: false
```

Comment:
Changing the default to false could be seen as a breaking change. What if instead we added a K8s version check to avoid the PSP even if this is true? But maybe it's not worth it since it's already been deprecated for so long.

Reply:
I added a note to the upgrade doc about the breaking change. I flip flopped about what would be the best approach.

Reply:
Can we get the k8s version from helm?

Reply:
We used to have a version check for the v1beta1 CRDs, see [this example](https://github.com/rook/rook/blob/release-1.7/cluster/charts/rook-ceph/templates/resources.yaml#L2)